src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.cpp

```diff
@@ -214,5 +206,41 @@ void BrgemmKernelExecutor::execute(const BrgemmKernelExecutor* executor, call_ar
     (*kernel->compiled_kernel)(&brgemm_p);
 }
 
+#ifdef SNIPPETS_DEBUG_CAPS
+BrgemmKernelReferenceExecutor::BrgemmKernelReferenceExecutor(ov::intel_cpu::MultiCacheWeakPtr kernel_cache, BrgemmKernelConfig config) :
+        BrgemmKernelExecutor(std::move(kernel_cache), std::move(config)) {
+}
+
+std::shared_ptr<BrgemmCompiledKernel> BrgemmKernelReferenceExecutor::compile_kernel(const BrgemmKernelConfig& c) const {
+    const auto& res = BrgemmKernelExecutor::compile_kernel(c);
```

Comment:
Do we really need to call `BrgemmKernelExecutor::compile_kernel` here?
Could you please elaborate a little bit? Probably I missed something but looks like we can create `std::shared_ptr<BrgemmCompiledKernel>` ourself... 🤔